src/components/home.js

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { FaSearch } from 'react-icons/fa';
+import { fetchCoins } from '../redux/currency/currency';
+import Display from './display';
+import './home.css';
+
+const Home = () => {
+  const dispatch = useDispatch();
+
+  const { coins, status } = useSelector((state) => state.coins);
+  const [search, setSearch] = useState('');
+  useEffect(() => {
+    if (status === null) {
+      dispatch(fetchCoins());
+    }
+  }, [status, dispatch]);
+
+  return (
+    <div>
+      <div className="imgDv">
+        <img className="main_image" src="https://images.pexels.com/photos/8919542/pexels-photo-8919542.jpeg?auto=compress&cs=tinysrgb&w=600" alt="" />
+      </div>
+      <div className="searchDiv">
+        <input
+          className="searchBar"
+          type="text"
+          value={search}
+          onChange={(e) => setSearch(e.target.value)}
+          placeholder="Search Here"
+        />
+        <FaSearch className="searchIcon" />
```

Comment:
It looks and works great! but for the design enhancement, please make sure to align it to the center of the page properly.
![image](https://user-images.githubusercontent.com/34779928/210870463-e42e5002-846e-4bfc-b435-f10c82d54f16.png)